TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -522,6 +524,11 @@ void SetLevelFuncCallback(TypeOrNil<LevelFunc> const & cb, std::string const & c
 
 }
 
+short Moveable::GetIndex() const
+{
+	return m_num;
```

Comment:
Why not `m_item->Index`?

Reply:
Because m_item is a protected variable from Moveable class, so it can't be called from external classes like CameraObject.

To can do that, I would have to make m_item (or more directly, m_num) into a public variable, but I though would be a better practice leave those variables for the internal class use, and add that GetIndex() for the external access.

PlayCamera needs that index provided from the moveable (sent from Lua by argument) and give it to the camera to use it as target item.